wordpress-mobile/WordPress-iOS
# [Stats Refresh] StatsPeriodStore: make use of the new API

This switches StatsPeriodStore to the new `StatsServiceRemoteV2` based API, also updating the appropriate ViewModels.

I found two more changes (grr.....) I'll need to make to the data model when working on this — 
* `Videos` endpoint doesn't return a media `ID`, just a URL
* we need to move the country-flag icons from WPStatsiOS module to WPiOS. This was outside of scope for this PR so I decided to leave it broken for now.

To test:

* Go to Stats
* Look at Period-type data
* Verify everything still works! (modulo the two broken things above)
